pkg/controller/node/node_controller_test.go

```diff
@@ -38,18 +37,21 @@ var expectedRequest = reconcile.Request{NamespacedName: types.NamespacedName{Nam
 const timeout = time.Second * 5
 
 func TestReconcile(t *testing.T) {
-	g := gomega.NewGomegaWithT(t)
 	instance := &corev1.Node{ObjectMeta: metav1.ObjectMeta{Name: "foo"}}
 
 	// Setup the Manager and Controller.  Wrap the Controller Reconcile function so it writes each request to a
 	// channel when it is finished.
 	mgr, err := manager.New(cfg, manager.Options{})
-	g.Expect(err).NotTo(gomega.HaveOccurred())
+	if err != nil {
+		t.Errorf("error creating new manager: %v", err)
+	}
 	c = mgr.GetClient()
 
 	recFn, requests := SetupTestReconcile(newReconciler(mgr))
-	g.Expect(add(mgr, recFn)).NotTo(gomega.HaveOccurred())
-	defer close(StartTestManager(mgr, g))
+	if err = add(mgr, recFn); err != nil {
```

Comment:
`err :=`

Reply:
`err` is already defined above. By redefining it just for the scope of the `if` statement, we would be producing a "shadowed" variable which linting with `go tool vet -shadow` would (did) complain about. I'm new to the project and am not yet familiar with the coding standards. I can certainly make this change, but I just want to be clear about why I did it. LMK